### PR TITLE
ASG UX update

### PR DESCRIPTION
**Pull request by Adam-Dupaski**

### CODE REVIEW

File: content/appstore/general/app-store-overview.md [hunk 1]
```diff
@@ -8,110 +8,104 @@ tags: ["app store", "widget", "connector", "module", "partner"]
...
+	* **Subscribe** – click this to [subscribe](#subscriptions) to a paid [platform-supported](app-store-content-support#category) app service
+	* <a name="try-for-free"></a>**Try for Free** – click this to start a 30-day free trial of the paid [platform-supported](app-store-content-support#category) component
+		* After you agree to the terms and conditions, your trial will be set up – click **Check Status** to go to the [My Subscriptions](#subscriptions) page, where you can click **Manage App Servcies** to go to the [Service Management](#service-management) page (you will also receive an email with a link to this page)
+		* If a [platform-supported](app-store-content-support#category) component has the **Premium** label, this means that after your 30-day free trial ends, you need to [subscribe](#subscriptions) to the component to use it
 
-{{% image_container width="300" %}}
-![](attachments/app-store-overview/content_detail_2.png)
-{{% /image_container %}}
+	{{% todo %}}[**Verify "Subscribe", "Try for Free", and "Premium" definitions**]{{% /todo %}}
```
Comment (Adam-Dupaski) [inline content/appstore/general/app-store-overview.md:77]: @viralypatel could you please verify the Subscribe, Try for Free, and Premium definitions above. I will also ask Ester.

File: content/appstore/general/share-app-store-content.md [hunk 2]
```diff
@@ -207,25 +181,21 @@ If an existing App Store component is assigned to a [user group](app-store-overv
 
 To update content that has already been published, follow these steps:
 
-1. Go to the information page for the content you want to update (for example, by selecting [My Content](/appstore/general/app-store-overview#content) in **My App Store** and finding the component you want to update).
-2.  Click **Create new draft version**:
-	
-	![](attachments/share-app-store-content/update_content_1.png)
-	
-	{{% alert type="info" %}}Only one draft version of a component can exist at a time, so when one draft version is in progress, another draft cannot be initiated. If there is a draft version in progress, click **View draft** to see it.
+1.  Find the component clicking **My Marketplace** > [My Content](/appstore/general/app-store-overview#content) and then **Manage** for the component to update
```
Comment (Adam-Dupaski) [inline content/appstore/general/share-app-store-content.md:184]: @viralypatel please verify the steps in this section for accuracy.

File: content/appstore/general/app-store-overview.md (hunk 1 above)
Comment (viralypatel) [reply content/appstore/general/app-store-overview.md:77]: @Adam-Dupaski Looks ok to me but would request confirming this one with Stuart/Ester. I'll loop them in to our conversation.

File: content/appstore/general/share-app-store-content.md (hunk 2 above)
Comment (viralypatel) [reply content/appstore/general/share-app-store-content.md:184]: @Adam-Dupaski For the "Publish Content" button might be good to add that the approval is required only for the first version of public content. Subsequent versions will not go through approval. Also, for private content there is no approval required.

File: content/appstore/general/app-store-overview.md (hunk 1 above)
Comment (Adam-Dupaski) [reply content/appstore/general/app-store-overview.md:77]: Have updated screenshot.

File: content/appstore/general/share-app-store-content.md (hunk 2 above)
Comment (Adam-Dupaski) [reply content/appstore/general/share-app-store-content.md:184]: Added note, thanks.

File: content/appstore/general/app-store-overview.md (hunk 1 above)
Comment (Adam-Dupaski) [reply content/appstore/general/app-store-overview.md:77]: Ester and Stuart, please verify.

**Pull request merged**